charts/telegraf-ds/templates/NOTES.txt

```diff
@@ -1,11 +1,11 @@
-To open a shell session in the container running Telegraf run the following:
+To open a shell session in the container running Telegraf, run the following:
 
-- kubectl exec -i -t --namespace {{ .Release.Namespace }} $(kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/name={{ include "telegraf.name" . }} -o jsonpath='{.items[0].metadata.name}') /bin/sh
+  kubectl exec -i -t --namespace {{ .Release.Namespace }} $(kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/name={{ include "telegraf.name" . }} -o jsonpath='{.items[0].metadata.name}') /bin/sh
 
-To tail the logs for a Telegraf pod in the Daemonset run the following:
+To view  the logs for a Telegraf pod in the Daemonset, run the following:
 
-- kubectl logs -f --namespace {{ .Release.Namespace }} $(kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/name={{ include "telegraf.name" . }} -o jsonpath='{ .items[0].metadata.name }')
+  kubectl logs -f --namespace {{ .Release.Namespace }} $(kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/name={{ include "telegraf.name" . }} -o jsonpath='{ .items[0].metadata.name }')
 
-To list the running Telegraf instances run the following:
+To list the running Telegraf instances:
```

Comment:
Should this read "running Telegraf pods"? I know this was here prior, but I'm wondering if we can use this as an opportunity to be more explicit.

Maybe we should also remove the `-w` or clarify what that does?

Reply:
@pierwill